backend/internal/controllers/utils.go

```diff
@@ -0,0 +1,12 @@
+package controllers
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// Sends data and status code for http
+func WriteResponse(rw http.ResponseWriter, statusCode int, body interface{}) {
```

Comment:
You can lowercase this to writeResponse, all the controllers are in the same package.